package/kernel/linux/modules/crypto.mk

```diff
@@ -1028,11 +1034,13 @@ define KernelPackage/crypto-sha1/mpc85xx
 endef
 
 ifndef CONFIG_TARGET_uml
+ifndef CONFIG_LINUX_6_18
 define KernelPackage/crypto-sha1/x86_64
   FILES+=$(LINUX_DIR)/arch/x86/crypto/sha1-ssse3.ko
-  AUTOLOAD+=$(call AutoLoad,09,sha1-ssse3)
+  AUTOLOAD+=$(call AutoLoad,09,!LINUX_6_18:sha1-ssse3)
```

Comment:
Simply add `DEPENDS:=@!LINUX_6_18` instead of `ifndef` or `!LINUX_6_18:`
here and in several similar places.